techniques/systemSettings/userManagement/userManagement/5.0/userManagement.st

```diff
@@ -249,7 +249,7 @@ bundle agent check_usergroup_user_parameters
 
       ## Added (Repaired)
       "@@userGroupManagement@@result_repaired@@${usergroup_directive_id[${usergroup_user_index}]}@@Users@@${usergroup_user_login[${usergroup_user_index}]}@@${g.execRun}##${g.uuid}@#The user ${usergroup_user_login[${usergroup_user_index}]} ( ${repname[${usergroup_user_index}]} ) has been added to the system"
-        ifvarclass => "!usermanagement_user_exists_${usergroup_user_index}.usermanagement_user_update_${usergroup_user_index}.usermanagement_login_add_${usergroup_user_index}_repaired";
+        ifvarclass => "usermanagement_user_exists_${usergroup_user_index}.usermanagement_user_update_${usergroup_user_index}.usermanagement_login_add_${usergroup_user_index}_repaired";
```

Comment:
Hum.
the whole idea of the !user_exists is to know if the user existed before or not (and it is repaired if user didn't exists before)
I guess the issue is that user didn't exist, user was create, user exist, do no reporting

I think a better fix would be to have a
"user_didn_t_exist" class which is not => userexists(...)